src/app/pages/charging-stations/charging-station/parameters/charging-station-parameters.component.html

```diff
@@ -77,6 +77,12 @@
           {{'chargers.force_inactive' | translate}}
         </mat-checkbox>
       </div>
+      <div *ngIf="currentType !== 'DC' || chargingStation?.chargePoints?.length === 0"
```

Comment:
@LucasBrazi06 
I excluded dc charging stations, because we have several additional things to handle in these cases. E.g. share power between all connectors property, which is a charge point property. 
My thought was that dc charging stations need to be connected on all three phases anyway and that there is no real use case, where a manual config is needed. Let me know if I am wrong here.
When the dc charging station has no template, it will be configurable anyway.

Reply:
> @LucasBrazi06
> I excluded dc charging stations, because we have several additional things to handle in these cases. E.g. share power between all connectors property, which is a charge point property.
> My thought was that dc charging stations need to be connected on all three phases anyway and that there is no real use case, where a manual config is needed. Let me know if I am wrong here.
> When the dc charging station has no template, it will be configurable anyway.

No, if it's manual, you don't have the possibility to do so.
Let the user enter the data per connector like for AC chargers and let responsible of setting these values.
Let him set it up like if there was no template, that's the goal of this setting in the end.